config/locales/cs.yml

```diff
@@ -0,0 +1,40 @@
+cs:
+  errors:
+    messages:
+      taken_in_past: bylo již použito v minulosti.
+      equal_to_current_password: se musí lišit od aktuálního hesla.
+      password_complexity:
+        digit:
+          one: musí obsahovat alespoň jednu číslici
+          other: musí obsahovat alespoň %{count} číslice
+        lower:
+          one: musí obsahovat alespoň jedno malé písmeno
+          other: musí obsahovat alespoň %{count} malé písmena
+        symbol:
+          one: musí obsahovat alespoň jedno interpunkční znaménko nebo symbol
+          other: musí obsahovat alespoň %{count} interpunkční znaménka nebo symboly
+        upper:
+          one: musí obsahovat alespoň jedno velké písmeno
+          other: musí obsahovat alespoň %{count} velké písmena
+  devise:
+    invalid_captcha: Chybná captcha.
+    invalid_security_question: Chybná odpověď na bezpečnostní otázku.
+    paranoid_verify:
+      code_required: Zadejte kód, který poskytla naše podpora
+    paranoid_verification_code:
+      show:
+        submit_verification_code: Odeslat ověřovací kód
+        verification_code: Ověřovací kód
+        submit: Odeslat
```

Comment:
Not sure if the translation is right here? It's coming across as "Send" on Google Translate, but the correct word is "Submit".

<img width="1281" alt="Screen Shot 2020-10-15 at 10 17 45 AM" src="https://user-images.githubusercontent.com/1635820/96142185-cae5a980-0ecf-11eb-82ce-b424ad99c9b5.png">

<img width="1281" alt="Screen Shot 2020-10-15 at 10 17 32 AM" src="https://user-images.githubusercontent.com/1635820/96142200-d042f400-0ecf-11eb-91d7-4d5a34b0961a.png">

Reply:
@oniofchaos This is a common word for this kind of action. The "předložit" or similar is used more like for official communication (between companies, state administration, etc).

Reply:
Ahhhhh that makes a lot of sense. Thanks!